src/cli/cmd/query/platform/platforms.go

```diff
@@ -2,3 +2,73 @@
 // SPDX-License-Identifier: Apache-2.0
 
 package platform
+
+import (
+	"github.com/cldcvr/terrarium/src/cli/internal/config"
+	"github.com/cldcvr/terrarium/src/cli/internal/utils"
+	"github.com/cldcvr/terrarium/src/pkg/db"
+	"github.com/cldcvr/terrarium/src/pkg/pb/terrariumpb"
+	"github.com/rotisserie/eris"
+	"github.com/spf13/cobra"
+)
+
+var (
+	cmd              *cobra.Command
+	flags            *terrariumpb.ListPlatformsRequest
+	flagOutputFormat string
+)
+
+func NewCmd() *cobra.Command {
+	cmd = &cobra.Command{
+		Use:     "platforms",
+		Aliases: []string{"p"},
+		Short:   "Query available platforms",
+		Long:    "command to query available platforms.",
```

Comment:
Shouldn't the Long have more context?

Reply:
Thanks @26christy for the review. I've updated the description. Please see if this makes now.
While at it, I also updated the Long description of `query modules` command.

Reply:
Looks good